test/unit/lib/plugins/aws/package/compile/events/apiGateway/lib/validate.test.js

```diff
@@ -235,6 +235,69 @@ describe('#validate()', () => {
     expect(() => awsCompileApigEvents.validate()).not.to.throw(Error);
   });
 
+  it('should throw when using a CUSTOM authorizer without an authorizer id', async () => {
```

Comment:
Let's move these two tests into the second `describe` at the bottom of the file that has only `runServerless` based tests